packages/vitest/src/node/reporters/renderers/utils.ts

```diff
@@ -258,3 +236,11 @@ export function withLabel(color: 'red' | 'green' | 'blue' | 'cyan' | 'yellow', l
 export function padSummaryTitle(str: string) {
   return c.dim(`${str.padStart(11)} `)
 }
+
+export function truncateString(text: string, maxLength: number): string {
+  if (stripVTControlCharacters(text).length <= maxLength) {
+    return text
+  }
+
+  return `${sliceAnsi(text, 0, maxLength - 1)}…`
+}
```

Comment:
`slice-ansi` is only required here. We could remove it if we didn't care about coloring when error messages' lines are truncated. Thoughts @hi-ogawa, @sheremet-va?

![image](https://github.com/user-attachments/assets/99708988-7419-4729-9cf2-bdcddbfb5bb5)

Reply:
I don't mind either way, but unless it's buggy or slow, I think It's good to have it.

Reply:
Not buggy or slow. I was mostly thinking about reducing install size, but maybe it's not worth it. Though this truncating is only done for these errors so it's not used most of the time. https://pkg-size.dev/slice-ansi

Reply:
I also don't mind

Reply:
I'll remove `slice-ansi` from here as its usage is quite an edge case.